src/libs/antares/study/parts/hydro/finallevelinflowsmodifyer.cpp

```diff
@@ -147,6 +148,18 @@ bool FinalLevelInflowsModifier::isActive()
             isValidLevel(initialReservoirLevel_);
 }
 
+// if the user specifies the final reservoir level, but does not specifies initial reservoir level
+// or uses wrong hydro options
+// we should inform the user that the final reservoir level wont be reached
+void FinalLevelInflowsModifier::logInfoFinLvlNotApplicable(uint year)
```

Comment:
To me, there is some inconsistencies here : 
- We log that the problem is **not** applicable, ok, but we don't skip or stop anything because of it. If the problem is not applicable, normally we should prevent something from happening, right ?
- **Applicable** is already taken by another property (see method **isApplicable**). We should distinguish the different form of applicability, or it starts to be unclear

Reply:
These info logs are as Florian suggested for crazy users. 
For example, final_level is set to 55% but initila_level is left at rand. 
In this case, we log out that for the specified MC-year and area, the final reservoir level won't be implemented (will be skipped). But we continue on implementation for the next year/are.
Logic won't be implemented for this year/area because "isApplicable_" is going to be false for that year/area.
This behavior should be distinguished from the one when some of the pre-checks fail and we break the simulation immediately. 
So we have situations where we exit Antares asap, or we skip the year/area and throw an info log.